shelf/bucket_update/utils.py

```diff
@@ -11,7 +11,18 @@ def update_search_index(bucket_config):
             bucket_config(schemas/search-bucket-update-config.json)
     """
     container = _create_container(bucket_config)
-    container.search_updater.run()
+    container.search_updater.update()
+
+
+def prune_search_index(bucket_config):
+    """
+        Prunes the search index for a specific bucket.
```

Comment:
I don't think this is sufficient. Again you might ask "what does this mean?". Thats why I asked for a description like I suggested before.

"Removes artifacts in the "search layer" that are not in the "cloud layer""